docs/UserGuide.md

```diff
@@ -202,6 +271,7 @@ Format: `find PREFIX KEYWORD [MORE_KEYWORDS]`
 - The search is case-insensitive. e.g `alex` will match `Alex`, `cs1231s` will match `CS1231S`.
 - We can apply multiple search filters to narrow down the search results.
 - Teaching assistants matching all the search parameters will be returned.
+- The search filters are applied to the original list of TAs, not the current view of TAs. 
```

Comment:
Maybe can mention if user wants to stack the filter, then need to write all the keyword in 1 command.